src/packages/src/xroad/ubuntu/generic/xroad-center.postinst

```diff
@@ -97,6 +101,8 @@ GRANT USAGE ON SCHEMA public to "${db_plain_user}";
 EOF
     fi
 
+    crudini --set "${db_properties}" '' password "${db_password}"
```

Comment:
There should be no need to modify the db.properties in case the database is not created by the install script  --> should be moved in the previous else block (lines 85-101).

Reply:
Yes I agree. Pushed the update.